i18n/es_ES.csv

```diff
@@ -160,12 +160,12 @@
 "Magento Enterprise Edition version","Versión Edición de empresa de Magento"
 "Quick reference guide","Guía de referencia rápida"
 "Log in to the ","Iniciar sesión en"
-"Buckaroo Payment Plaza","Payment Plaza de Buckaroo"
+"Buckaroo Buckaroo Plaza","Buckaroo Plaza de Buckaroo"
 " to collect the following security settings:","para recopilar los siguientes ajustes de seguridad:"
 "Support","Asistencia"
 "This extension is developed by Buckaroo.","Esta extensión la ha desarrollado Buckaroo."
 "Extension basic configuration and account information","Configuración básica de la extensión e información de la cuenta"
-"Buckaroo Payment Plaza:","Payment Plaza de Buckaroo:"
+"Buckaroo Buckaroo Plaza:","Buckaroo Plaza de Buckaroo:"
```

Comment:
same

Reply:
**This should be:**
"Buckaroo Plaza:","Buckaroo Plaza:"